app/models/edition/workflow.rb

```diff
@@ -2,9 +2,11 @@ module Edition::Workflow
   extend ActiveSupport::Concern
   include DateValidation
 
-  module ClassMethods
+  STATES = %w[draft published scheduled superseded awaiting_2i].freeze
+
+  class_methods do
     def valid_state?(state)
-      %w[draft published scheduled superseded].include?(state)
+      STATES.include?(state)
```

Comment:
This is a good idea, but I think rather than having the states in a constant and setting them up as below, I'd rather the state_machine was the source of truth. You can then use `Edition.available_states` in the validator.